starport_template/lib/pages/transfer_asset_page.dart

```diff
@@ -16,7 +20,7 @@ class TransferAssetPage extends StatefulWidget {
 
 class _TransferAssetPageState extends State<TransferAssetPage> {
   double amount = 0.0;
-  double fee = 0.0;
+  double fee = 0.02;
```

Comment:
since you're using this `0.02` value in multiple places, maybe we could export it as a static const to, lets say, `WalletsStore`?